keps/sig-instrumentation/20181106-kubernetes-metrics-overhaul.md

```diff
@@ -177,10 +173,63 @@ These old metrics will be deprecated in v1.14 and coexist with the new replaceme
 
 The release target of removing the deprecated metrics is v1.15.
 
+Prior to removing deprecated metrics, we will attend appropriate community meetings (i.e. SIG Node) to provide sufficient notice.
+
 ## Graduation Criteria
 
 All metrics exposed by components from kubernetes/kubernetes follow Prometheus best practices and (nice to have) tooling is built and enabled in CI to prevent simple violations of said best practices.
 
 ## Implementation History
 
-Multiple pull requests have already been opened, but not merged as of writing of this document.
+As of release 1.17, this KEP is considered fully implemented.
```

Comment:
It took me like an hour to dig up all these PRs and update the KEP so I hope this will be appreciated :grin: